packages/design-system-docs/src/pages/components/FilterTag/filtertag.example.html

```diff
@@ -0,0 +1,23 @@
+<div class="ds-c-filter-tag">
+  <button class="ds-c-filter-tag__button" id="Example filter tag-tag">
+    <span class="ds-u-visibility--screen-reader">Remove Example Filter Tag </span
```

Comment:
I think we can remove this screen reader span and instead add `aria-label="Remove Example Filter Tag"` to the span with the clear icon on it